client_dashboard/common/src/commonMain/kotlin/org/thechance/common/data/remote/model/RestaurantDto.kt

```diff
@@ -0,0 +1,11 @@
+package org.thechance.common.data.remote.model
+
+data class RestaurantDto(
+    val id: String,
+    val restaurantName: String,
```

Comment:
```suggestion
    val name: String,
```
no need to mention the restaurant name in the variable the data class already mentions it